docs/_data/prague-2021-schedule.yaml

```diff
@@ -0,0 +1,79 @@
+writing_day:
+- time: '10:00'
```

Comment:
This looks a bit later than we usually started writing day, but I'm not actually mad at it... Maybe we can trim the time between opening the platform and doing the introduction to 30min? We tried something like that at some point and it was good....